lib/susedistribution.pm

```diff
@@ -168,5 +166,55 @@ sub become_root() {
     send_key('ctrl-l');
 }
 
+# initialize the consoles needed during our tests
+sub activate_consoles {
+    my ($self) = @_;
+    if (get_var('BACKEND', 'qemu')) {
```

Comment:
all is very nice but at this location I would propose to use a hashmap and not procedural code if/else to lookup the corresponding console and such from a map. Then, the corresponding map could later on be moved easily into the os-autoinst as "backend".